crates/blockchain-tree/src/chain.rs

```diff
@@ -28,8 +28,8 @@ use std::{
     time::Instant,
 };
 
-/// A chain if the blockchain tree, that has functionality to execute blocks and append them to the
-/// it self.
+/// A chain if the blockchain tree, that has functionality to execute blocks and append them to
```

Comment:
Also just noticed... is this supposed to be "A chain *of* the blockchain tree"?

Reply:
I believe this should be "A sidechain in the blockchain tree"

Reply:
Ahh yes. Will make that change too.